diff -u Makefile.am Makefile.am
--- Makefile.am
+++ Makefile.am
@@ -12,6 +12,7 @@
 test_chealpix_LDADD = libchealpix.la
 
 AM_CPPFLAGS = $(CFITSIO_CFLAGS)
+AM_LDFLAGS = -no-undefined
 
 pkgconfigdir = $(libdir)/pkgconfig
 nodist_pkgconfig_DATA = @PACKAGE_NAME@.pc
diff -u configure.ac configure.ac
--- configure.ac
+++ configure.ac
@@ -37,7 +37,7 @@
 dnl
 dnl Linking against libcfitsio requires libm too.
 dnl
-CFITSIO_LIBS="$CFITSIO_LIBS -lm"
+CFITSIO_LIBS="$CFITSIO_LIBS"
 dnl
 dnl With pkg-config < 0.24, PKG_CHECK_MODULES does not AC_SUBST the _CFLAGS
 dnl and _LIBS variables.
